### PR TITLE
minimap2 2.0

### PR DESCRIPTION
**Pull request by tseemann**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

### CODE REVIEW

**Comment by tseemann** [comment]
```
homebrew/science/minimap2:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
  * GitHub repository too new (<30 days old)
```

We really need an exception for this for Heng Li's software!

What should I do?

**Comment by tseemann** [comment]
Update: 
The author would like to wait another week before making a release: https://github.com/lh3/minimap2/issues/5

**Comment by jonchang** [comment]
This might work now that https://github.com/Homebrew/brew/pull/2924 has been merged.

**Comment by jonchang** [comment]
@BrewTestBot test this please

File: minimap2.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Minimap2 < Formula
+  desc "Fast mapping and alignment for long noisy reads"
+  homepage "https://github.com/lh3/minimap2"
+  # tag "bioinformatics"
+
+  url "https://github.com/lh3/minimap2.git",
+    :revision => "5c4d040b13f6aa714696605f3cf618ad5b1f934e"
+  version "r191"
```
Comment (jonchang) [inline minimap2.rb]: Should this link to `https://github.com/lh3/minimap2/archive/r191.tar.gz`?

Comment (tseemann) [inline minimap2.rb]: he didnt makle a release at the time, i asked him to. looks like he has now!

**Comment by tseemann** [comment]
@BrewTestBot test this please

**Comment by tseemann** [comment]
@jonchang are you able to assist?  this tool is in great demand right now.

**Comment by jonchang** [comment]
I restarted the circle build since it timed out.

**Comment by jonchang** [comment]
It built properly on travis so circle should probably succeed as well.

**Comment by jonchang** [comment]
@tseemann you'll need to rebase your branch on master. `git checkout minimap2_r187 && git rebase origin/master` should do it.

**Comment by tseemann** [comment]
thanks for your help again. i struggle to interpret any of the new circle/travis stuff.

**Comment by tseemann** [comment]
Will do with latest.

**Pull request closed without merging**